sql/executor.go

```diff
@@ -345,3 +350,13 @@ func (p parameters) Arg(name string) (parser.Datum, bool) {
 		panic(fmt.Sprintf("unexpected type %T", t))
 	}
 }
+
+func getTimeZone(s Session) (*time.Location, error) {
```

Comment:
I think we much prefer pure functions that take in parameters by value.


On Tue, Sep 29, 2015 at 9:27 AM Spencer Kimball <notifications@github.com>
wrote:

> In sql/executor.go
> <https://github.com/cockroachdb/cockroach/pull/2698#discussion_r40670225>:
>
> > @@ -345,3 +350,13 @@ func (p parameters) Arg(name string) (parser.Datum, bool) {
> >  		panic(fmt.Sprintf("unexpected type %T", t))
> >  	}
> >  }
> > +
> > +func getTimeZone(s Session) (*time.Location, error) {
>
> Strange not to make this a function on a Session object.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2698/files#r40670225>.
>